xbmc/addons/AddonInstaller.h

```diff
@@ -23,6 +23,36 @@ namespace ADDON
 
 class CAddonDatabase;
 
+enum class BackgroundJob
+{
+  IS_BACKGROUND,
+  NOT_BACKGROUND,
+};
+
+enum class ModalJob
+{
+  IS_MODAL,
+  NOT_MODAL,
+};
+
+enum class AutoUpdateJob
+{
+  IS_AUTOUPDATE,
+  NOT_AUTOUPDATE,
+};
+
+enum class DependsInstallJob
```

Comment:
To be consistent maybe `DependencyJob` would be more correct with `IS_DEPENDENCY` and `NOT_DEPENDENCY` as the members.

Reply:
true